### PR TITLE
missing file __Attributions.md

### PR DESCRIPTION
**Pull request by xno**

### CODE REVIEW

**Comment by crane-denny** [comment]
https://github.com/Courseplay/Courseplay_FS22/pull/219/commits/77b15cdbae771c9a8d5a900c594907eee396647a#diff-874e1a159c0dffcaf38d1d999e49fedf04781448a627a9c1c0f6b3a6b64f6e4aR10

and 

https://github.com/Courseplay/Courseplay_FS22/pull/219/commits/77b15cdbae771c9a8d5a900c594907eee396647a#diff-874e1a159c0dffcaf38d1d999e49fedf04781448a627a9c1c0f6b3a6b64f6e4aR14

are the same languages. :)

**Comment by xno** [comment]
Ok. Merged to dutch

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ ___
 ___
 
 ## Attributions
-Please see [`__Attributions.md`](https://github.com/Courseplay/Courseplay_FS22/blob/master/__Attributions.md) for more information.
+Please see [`__Attributions.md`](https://github.com/Courseplay/Courseplay_FS22/master/__Attributions.md) for more information.
```
Comment (Thodor12) [inline README.md:60]: Github supports relative links in their Markdown files, currently you are attempting to point to an absolute path, along with that you are pointing to the `master` branch, which this repository is not using, they are using `main` which is the default for Github.
If you change this to
```md
Please see [`__Attributions.md`](__Attributions.md) for more information.
```
It should work just fine

**Review by Thodor12 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (xno) [reply README.md]: Thanks for the suggestion.
Run to apply the change and post a new commit.

**Comment by pvaiko** [comment]
I don't know why this file exists, why can't we just put this information in the README directly? First I thought this is some GitHub feature, triggering some special processing but I can't find any reference to it. Do we really need this?

**Pull request closed without merging**